src/plugins/initiatives/initiativesDirectory.js

```diff
@@ -60,3 +69,37 @@ export function fromJSON(j: Compatible<any>): InitiativeFile {
 export function toJSON(m: InitiativeFile): Compatible<InitiativeFile> {
   return toCompat(COMPAT_INFO, m);
 }
+
+/**
+ * When provided with the tracker NodeAddressT of an InitiativeFile this extracts
+ * the URL from it. Or null when the address is not for an InitiativeFile.
+ */
+export function initiativeFileURL(trackerAddress: NodeAddressT): string | null {
+  if (!NodeAddress.hasPrefix(trackerAddress, initiativeFilePrefix)) {
+    return null;
+  }
+
+  const parts = NodeAddress.toParts(trackerAddress);
+  const remoteUrl = parts[3];
+  const fileName = parts[4];
+  return `${remoteUrl}/${fileName}`;
+}
+
+// Creates the tracker NodeAddressT for an InitiativeFile.
+export function _trackerAddress(
+  remoteUrl: string,
+  fileName: string
+): NodeAddressT {
+  return NodeAddress.append(initiativeFilePrefix, remoteUrl, fileName);
+}
+
+// Creates the Initiative NodeAddressT for an InitiativeFile.
+export function _initiativeAddress(
+  remoteUrl: string,
+  fileName: string
+): NodeAddressT {
+  return NodeAddress.append(
+    initiativeNodeType.prefix,
+    ...NodeAddress.toParts(_trackerAddress(remoteUrl, fileName))
+  );
+}
```

Comment:
hmm, i'm confused. what is the difference between an initiative address and a tracker address, and why do we have both? If we do keep both around, the rationale should be documented in the code here, but my preference is to only have one address per initiative.

Reply:
Now depending on #1647 to remove trackers.